### PR TITLE
Feature/stlouis scrapers

### PR DESCRIPTION
**Pull request by jesseilev**
Created a Jurisdiction for St Louis with one organization, the St Louis Board of Aldermen, and implemented a basic Person scraper and Bill scraper.

### CODE REVIEW

File: st_louis/__init__.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+class StLouis(Jurisdiction):
+    division_id = "ocd-division/country:us/state:mo/place:st_louis"
+
+    # FIXME
+    # Do we need jurisdiction_id? pupa did not auto-generate this field 
+    # like division_id, I made it myself.
+    # And is this the correct id? or should it be 
+    # .../place:st_louis/council ? or .../place:st_louis/government ?
+    jurisdiction_id = "ocd-jurisdiction/country:us/state:mo/place:st_louis"
```
Comment (fgregg) [inline st_louis/__init__.py]: You shouldn't need to do this. just declare division id and classification like you do.

File: st_louis/bills.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+from pupa.scrape import Scraper
+from pupa.scrape import Bill
+from pupa.scrape import BaseBillScraper
+from .utils import Utils
+import time
+
+
+class StLouisBillScraper(Scraper):
+
+	def scrape(self):
```
Comment (fgregg) [inline st_louis/bills.py]: Could you always indent with four spaces. Looks like you are doing that in the previous files.

**Comment by fgregg** [comment]
Nice! I'll take a deeper look monday.

**Comment by fgregg** [comment]
When I run `pupa update st_louis` I get 

this error

```
20:25:46 INFO pupa: save bill BB 238 in 2010-2011 as bill_a61f35c6-7c49-11e5-bbfb-0024d703f52c.json
Traceback (most recent call last):
  File "/home/fgregg/public/opencivicdata/.env/bin/pupa", line 9, in <module>
    load_entry_point('pupa==0.5.0', 'console_scripts', 'pupa')()
  File "/home/fgregg/public/opencivicdata/repos/pupa/pupa/cli/__main__.py", line 71, in main
    subcommands[args.subcommand].handle(args, other)
  File "/home/fgregg/public/opencivicdata/repos/pupa/pupa/cli/commands/update.py", line 242, in handle
    report['scrape'] = self.do_scrape(juris, args, scrapers)
  File "/home/fgregg/public/opencivicdata/repos/pupa/pupa/cli/commands/update.py", line 141, in do_scrape
    report[scraper_name] = scraper.do_scrape(**scrape_args)
  File "/home/fgregg/public/opencivicdata/repos/pupa/pupa/scrape/base.py", line 101, in do_scrape
    for obj in self.scrape(**kwargs) or []:
  File "/home/fgregg/public/opencivicdata/repos/scrapers-us-municipal/st_louis/bills.py", line 25, in scrape
    yield self.scrape_bill(bill_url, bill_id, session_id)
  File "/home/fgregg/public/opencivicdata/repos/scrapers-us-municipal/st_louis/bills.py", line 51, in scrape_bill
    summary = data_table.xpath(self.bill_table_query("Summary"))[0]
IndexError: list index out of range
```

File: st_louis/bills.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+		for session in self.jurisdiction.legislative_sessions:
+			session_id = session["identifier"]
+			session_url = self.bill_session_url(session_id)
+			page = Utils.lxmlize(session_url)
```
Comment (fgregg) [inline st_louis/bills.py]: If you used defined lxmlize like this https://github.com/opencivicdata/scrapers-us-municipal/blob/master/chicago/legistar.py#L18, where the requests are using the `Scrapers` class method than you get a lot of things for free, including caching.

**Comment by jesseilev** [comment]
As a quick-fix to the error you got while scraping, I placed the offending line inside a `try/catch` block.

I also refactored the `lxmlize` method as you suggested.

**Comment by jesseilev** [comment]
@fgregg Any luck with the changes I made?

**Comment by fgregg** [comment]
Thanks! We'll add it to our nightly scrape, and I'll let you know when the data is available through the API.

**Comment by jesseilev** [comment]
Awesome thanks!

**Comment by fgregg** [comment]
@jesseilev your data is available at  http://ocd.datamade.us/ocd-organization/c0bf8d80-b75e-43ce-af38-384a7ef12a7f/

**Pull request merged**